src/database.py

```diff
@@ -70,6 +72,15 @@ class User(Base):
     bans_as_seller = relationship(
         "BannedPair", back_populates="seller", foreign_keys="[BannedPair.seller_id]"
     )
+    seller = relationship(
```

Comment:
This seems wrong. This means that `user.seller` accesses the chat rooms where this user is a seller. Maybe `chats_as_seller` or sth liddat?

Reply:
Ditto the bottom 2 variables